tutorials/SB3/test/test_sb3_action_mask.py

```diff
@@ -30,7 +30,7 @@
 MEDIUM_ENVS = [
     leduc_holdem_v4,  # with 10x as many steps it gets higher total rewards (9 vs -9), 0.52 winrate, and 0.92 vs 0.83 total scores
     hanabi_v5,  # even with 10x as many steps, total score seems to always be tied between the two agents
-    tictactoe_v3,  # even with 10x as many steps, agent still loses every time (most likely an error somewhere)
+    tictactoe_v4,  # even with 10x as many steps, agent still loses every time (most likely an error somewhere)
```

Comment:
So how does this new env fare here? Should the comment be updated? Funny that this would be a sign of the underlying env having issues, I assumed it was my tutorial code being bad (probably both to be fair)

Reply:
It trains much better. It needs a little more time than the easy step limit. At 4x as many steps, it passed 5 times in a row for me. I can update that comment/code tomorrow

Reply:
Awesome super appreciate your testing this